docs/releases/versioning-policy.md

```diff
@@ -0,0 +1 @@
+The Kubeflow Pipelines versioning follows the [semantic versioning policy](https://semver.org/). The Kubeflow Pipelines versions are in the format of vX.Y.Z, where X is the major version, Y is the minor version, and Z is the patch version. The major version is incremented for API/feature changes that are new and not backward-compatible; the minor version is incremented for API/feature changes that are new and backward-compatible; and the patch version is for bug fixes between two minor versions. The Kubeflow Pipelines vX.Y.Z refers to the version (git tag) of the released Kubeflow Pipelines and it versions all servers and the SDK of the Kubeflow Pipelines. Moreover, if the version number includes an appendix -rcN, where N is a number, the appendix indicates a release candidate, which is a pre-release version of an upcoming release.
```

Comment:
nit: our versions are in the format of `X.Y.Z`, without the "v" prefix.

RCs are formatted as `X.Y.Z-rc.N`, that's different from kubeflow.

Reply:
done